cves/CVE-2003-0020.yml

```diff
@@ -31,7 +31,13 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  Terminals are susceptible to exploits in the form of harmful inputs. Take,
```

Comment:
I feel it is important to specify that only certain terminal emulators allow for the more exploitable escape sequences such as command execution, window title and screen dumping. Additionally, exploit to me implies a bug, these escape sequences were purposeful features that are being misused.

Reply:
According to the first set of notes in SWEN-331, an exploit is "a piece of software, a chunk of data, or a sequence of commands that takes advantage of a vulnerability in an effort to cause unintended or unanticipated behavior (i.e., **maliciously using a vulnerability**)." If CVE-2003-0020 is considered a vulnerability, then the escape sequences that cause "unanticipated behavior" must be considered an exploit, according to this definition.

I pushed a commit that clarifies that not all terminal emulators are susceptible to this issue. Thanks for the feedback.